src/styles/reset.less

```diff
@@ -1,5 +1,7 @@
+@import "./vars.less";
```

Comment:
might be better to define the css variables in a separate file

also, would prob be better to have e.g. light mode variables be default, then override with dark mode selector